modules/5_install/install.tf

```diff
@@ -265,7 +265,9 @@ resource "null_resource" "powervs_config" {
 
 resource "null_resource" "upgrade" {
     depends_on = [null_resource.install, null_resource.powervs_config]
-    count      = var.upgrade_version != "" ? 1 : 0
+    triggers = {
```

Comment:
Count is not needed here if I want to deploy a cluster without upgrade?

Reply:
Fixed.